articles/service-fabric/cluster-security-certificate-management.md

```diff
@@ -25,35 +25,35 @@ As we've seen in the [companion article](cluster-security-certificates.md), a ce
 
 ## Roles and entities involved in certificate management
 The security approach in a Service Fabric cluster is a case of "cluster owner declares it, Service Fabric runtime enforces it". By that we mean that almost none of the certificates, keys, or other credentials of identities participating in a cluster's functioning come from the service itself; they are all declared by the cluster owner. Furthermore, the cluster owner is also responsible for provisioning the certificates into the cluster, renewing them as needed, and ensuring the security of the certificates at all times. More specifically, the cluster owner must ensure that:
-  - certificates declared in the NodeType section of the cluster manifest can be found on each node of that type, according to the [presentation rules](cluster-security-certificates.md#presentation-rules)
-  - certificates declared above are installed inclusive of their corresponding private keys
-  - certificates declared in the presentation rules should pass the [validation rules](cluster-security-certificates.md#validation-rules) 
+  - Certificates declared in the NodeType section of the cluster manifest can be found on each node of that type, according to the [presentation rules](cluster-security-certificates.md#presentation-rules)
+  - Certificates declared above are installed with their corresponding private keys included.
+  - Certificates declared in the presentation rules should pass the [validation rules](cluster-security-certificates.md#validation-rules) 
 
-Service Fabric, for its part, assumes the responsibilities of:
-  - locating/finding certificates matching the declarations in the cluster definition  
-  - granting access to the corresponding private keys to Service Fabric-controlled entities on a 'need' basis
-  - validating certificates in strict accordance with established security best-practices and the cluster definition
-  - raising alerts on impending expiration of certificates, or failures to perform the basic steps of certificate validation
-  - validating (to some degree) that the certificate-related aspects of the cluster definition are met by the underlying configuration of the hosts 
+Service Fabric, for its part, assumes the following responsibilities:
+  - Locating certificates matching those declared in the cluster definition
+  - Granting access to the corresponding private keys to Service Fabric-controlled entities on a 'need' basis
+  - Validating certificates in strict accordance with established security best-practices and the cluster definition
+  - Raising alerts on impending expiration of certificates, or failures to perform the basic steps of certificate validation
+  - Validating (to some degree) that the certificate-related aspects of the cluster definitions are met by the underlying configuration of the hosts 
```

Comment:
>ertificate-related aspects of the cluster definitions [](http://example.com/codeflow?start=42&length=53)

here, too, I believe the singular use of "definition" was correct; the intent of this line is to say that the SF runtime operating in a given cluster is responsible for validating the definition of _that_ cluster against the state of each machine/node which joined that cluster.